server/src/main/java/org/elasticsearch/index/seqno/RetentionLeaseSyncAction.java

```diff
@@ -91,10 +91,10 @@ public RetentionLeaseSyncAction(
             RetentionLeaseSyncAction.Request::new,
             RetentionLeaseSyncAction.Request::new,
             new ManagementOnlyExecutorFunction(threadPool),
-            PrimaryActionExecution.RejectOnOverload,
+            PrimaryActionExecution.Force,
```

Comment:
I don't believe this change is necessary on the `RetentionLeaseBackgroundSyncAction` because it's not a `TransportWriteAction`, and therefore `checkPrimaryLimits` (where the indexing pressure check is done) is a no-op.

Reply:
I wonder if we even need to make any changes to `RetentionLeaseBackgroundSyncAction`. I don't believe its failure will have the same consequences as that of `RetentionLeaseSyncAction`.